### ISSUE TITLE
SAS traces are confused by high numbers of sproutlet selection logs

### ISSUE THREAD

**Issue by benritchie** [Metaswitch/sprout#1690]
#### Symptoms
The below is an example High Level Event flow for one of our Live Test calls for a project I work on that uses sprout:
![image](https://cloud.githubusercontent.com/assets/8005676/22076477/233a9972-dda8-11e6-9ddb-866f5b32d2cc.png)

Several things I think we can do better here.  
### we have duplicate logs.  
L3 added sproutlet selection logs at low level here:
https://github.com/Metaswitch/clearwater-sas-resources/pull/113/files
and Ellie added them at high level here: https://github.com/Metaswitch/clearwater-sas-resources/pull/95
(Note, these are not viewable in the above timeline that is at a high level.  They look like this - Engineering level view:)
![image](https://cloud.githubusercontent.com/assets/8005676/22076720/39fdf996-dda9-11e6-8703-20afeb33a686.png)

We should delete one of those sets.  (incentives: reduce SAS resource usage, and tidy up the SAS view)

### Too much spam at high level
I  suggest that high level should have only a high level trace of what sproutlet we are entering.  
Personally, - I think the L3 logs are the clearest logs to provide these, so I'd upgrade these to L80.  

I suggest we then downgrade (where still useful) or delete (where duplicates), the rest of the SAS logs.  

### the No Sproutlet selected log.  
To me, this reads as an error.  However its not, I think we always get it just before forwarding a message to another box, to say "nothing more to do here".  Can we reword this to make it clearer, maybe "No more sproutlets to invoke, sending message?"

#### Release and environment 
around release 112.  (I don't know the exact build number, but this behaviour hasn't changed in a whle)

#### Steps to reproduce
should be visible on any call.  Let me know if you want to see this particular one

**Comment by benritchie** [comment]
I think this is a regression in V10, so my steer is it needs fixing in V10.

**Comment by eleanor-merry** [comment]
Similar issue to https://github.com/Metaswitch/sprout/issues/1650 (with some unique bits in each issue)

**Comment by benritchie** [comment]
I'm told I should have categorised this, so doing that.

**Comment by benritchie** [comment]
is this likely to be fixed in V10?

**Comment by philcrocker** [comment]
I agree not a major issue.  I don't want to fix in V10 - too late.

**Comment by benritchie** [comment]
I appreciate it's now too late for V10, but I would like to push for this to be fixed in V11.  - its a recent regression, and it clutters SAS significantly.

**Comment by benritchie** [comment]
also, I'm not a bit confused by the clearwater issue tagging - this is a critical issue, which I thought meant it was a V10 ship-stopper.  what have I missed?

**Comment by philcrocker** [comment]
A decision was made for V10 to just fix "major" issues, so a reduction in quality bar from previous.  Critical issues will all be under consideration for fixing in V11.
